atuin/src/command/client/search/interactive.rs

```diff
@@ -571,20 +571,17 @@ impl State {
                 Span::raw(": edit"),
                 Span::raw(", "),
                 Span::styled("<enter>", Style::default().add_modifier(Modifier::BOLD)),
-                Span::raw(": execute"),
+                Span::raw(": run"),
                 Span::raw(", "),
-                Span::styled("<ctrl-r>", Style::default().add_modifier(Modifier::BOLD)),
```

Comment:
useful, but not crucial to go here imo. I'd like to replace this with a help popup anwyay